walk-in-interview/src/main/webapp/log-in/business/script.js

```diff
@@ -55,4 +62,51 @@ backButton.addEventListener('click', (_) => {
 const submitButton = document.getElementById('submit');
 submitButton.addEventListener('click', (_) => {
```

Comment:
Do we need to temporarily disable this button while logging in to avoid clicking multiple times?

Reply:
As discussed offline, disable the button before line 65, enable the button within the block of success case and failure case